SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -201,8 +202,17 @@ public SingularityRequestParent bounce(@ApiParam("The request ID to bounce") @Pa
 
     requestManager.bounce(requestWithState.getRequest(), System.currentTimeMillis(), JavaUtils.getUserEmail(user), message);
 
+    SingularityBounceRequest defaultBounceRequest = bounceRequest.or(SingularityBounceRequest.defaultRequest());
+    if (!defaultBounceRequest.getDurationMillis().isPresent()) {
+      final Long durationMillis = TimeUnit.MINUTES.toMillis(configuration.getDefaultBounceExpirationMinutes());
```

Comment:
this is a nitpick, but you should utilize primitive types wherever possible (i.e. `long` instead of `Long`)